samples/BrepBooleanOperation/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Rhino.Compute;
+using Rhino.Geometry;
+
+namespace BrepBooleanOperation
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ComputeServer.AuthToken = Rhino.Compute.AuthToken.Get();
+
+            var cube = new Box(new BoundingBox(-0.5, -0.5, -0.5, 0.5, 0.5, 0.5));
+            var sphere = new Sphere(new Point3d(0.5, 0.5, 0.5), 0.8);
+
+            Brep[] cubeBrep = new Brep[] { cube.ToBrep() };
+            Brep[] sphereBrep = new Brep[] { sphere.ToBrep() };
+
+            const double tolerance = 1.0e-8;
+            {
```

Comment:
I just created the scoped for readability and because `allBreps` only needed for the union calculation. I like the idea to show every possible operation syntax in one example, but if you would like, I can keep only one (the difference for example).

Reply:
I like that you’ve included the different boolean operations, just wasn’t sure whether specifically scoping each one was necessary since it’ll all go out of scope at the end of the function anyway. Just trying to keep things simple :) Feel free to argue against!

Reply:
It's not worth an argument, I removed the scopes :)

By the way I like to use scopes to make clear I want to use a variable very locally so I can make sure no other codes in the function can access it. But you are right, in a short function like this it is not relevant.